include/rocksdb/advanced_options.h

```diff
@@ -953,6 +955,12 @@ struct AdvancedColumnFamilyOptions {
   // Dynamically changeable through the SetOptions() API
   int blob_file_starting_level = 0;
 
+  // If non-NULL use the specified cache for blobs.
+  // If NULL, rocksdb will not use a blob cache.
+  //
+  // Default: nullptr (disabled)
```

Comment:
May be add a comment that feature is in progress or currently not supported.

Reply:
done